### ISSUE TITLE
Next steps of development?

### ISSUE THREAD

**Issue by djuelg** [yydcdut/RxMarkdown#37]
Hey there!

I just would like to know your plan/schedule with this library.

* Do you intend to get out of beta (and when roughly) ?
* When do you plan to release 0.1.1 ?

Greetings,
djuelg

**Comment by stefan-niedermann** [comment]
Yeah, @yydcdut some kind of estimation when the next release will happen would be awesome :-)

In general it would be great if you could use GitHub Milestones, so all the users could get an impression about how many features / bug fixes are already finished and which might need help or Pull Requests to get a new version finished :)

Keep on the great work :rocket:

**Comment by yydcdut** [comment]
OK, thank you~

**Comment by stefan-niedermann** [comment]
@yydcdut when do estimate to release 0.1.3? #50 is a blocking issue which makes this version unusable unfortunately :-/
